test/jdk/java/net/httpclient/ExpectContinueTest.java

```diff
@@ -207,6 +213,30 @@ public void handle(HttpTestExchange exchange) throws IOException {
         }
     }
 
+    static class ForcePostHandler implements HttpTestHandler {
+        @Override
+        public void handle(HttpTestExchange exchange) throws IOException {
+            try (InputStream is = exchange.getRequestBody()) {
+                err.println("Server reading body");
+                is.readAllBytes();
+                err.println("Server send 200 (length=0)");
+                exchange.sendResponseHeaders(200, 0);
+            }
+        }
+    }
+
+    static class ForcePostHandlerHttp2 implements HttpTestHandler {
+        @Override
+        public void handle(HttpTestExchange exchange) throws IOException {
+            try (InputStream is = exchange.getRequestBody()) {
+                err.println("Server reading body inside the force Post");
+                is.readAllBytes();
+                err.println("Server send 200 (length=0) in the force post");
+                exchange.sendResponseHeaders(200, 0);
+            }
+        }
+    }
```

Comment:
These two seem to be identical: can't we just use ForcePostHandler both for HTTP/1.1 and HTTP/2?